time_weighted_average_price/attest_fn_call/iteration-call.json.template

```diff
@@ -4,13 +4,13 @@
     "input": {
         "coin_id": "polygon-pos-bridged-weth-polygon-pos",
         "num_samples": 3,
-        "eAttest": VAR_EATTEST,
-        VAR_TATTEST
+        "eAttest": PREV_ENCLAVE_ATTESTATION,
+        "tAttest": PREV_TRANSITIVE_ATTESTATION,
```

Comment:
I'm not sure why the eAttest field was always included but the tAttest field was conditional, where the end result was the same. I updated this file so they are treated the same.

Reply:
I also updated the naming since I was already changing this part everywhere to something slightly more self-describing